rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -99,6 +101,32 @@ def _toJSON(mol):
   return ""
 
 
+def _toHTML(mol):
+  if _canUse3D and ipython_3d and mol.GetNumConformers():
+    return _toJSON(mol)
+  props = mol.GetPropsAsDict()
+  if not ipython_showProperties or not props:
+    return _toSVG(mol)
+  if mol.HasProp('_Name'):
+    nm = mol.GetProp('_Name')
+  else:
+    nm = ''
+  res = []
+  if not ipython_useSVG:
+    png = Draw._moltoimg(mol, molSize, [], nm, returnPNG=True, drawOptions=drawOptions)
+    png = base64.b64encode(png)
+    res.append(f'<tr><td colspan=2><image src="data:image/png;base64,{png.decode()}"></td></tr>')
+  else:
+    svg = Draw._moltoSVG(mol, molSize, [], nm, kekulize=kekulizeStructures, drawOptions=drawOptions)
+    res.append(f'<tr><td colspan=2>{svg}</td></tr>')
+
+  for pn, pv in props.items():
```

Comment:
This might be a step too far for this PR, but having formatters available might be useful I can see some floats being wicked long.

Reply:
yeah, I thought about that and we'd definitely need a bunch of formatting options if there were some kind of `showMoleculeWithProperties()` function, but this is really intended to be a quick view.

If it ends up being a problem, we can be a bit cleverer when outputting the results for doubles and make sure that you don't end up with 15 digits.